.github/workflows/motoko-send-http-get-skip.yml

```
@@ -0,0 +1,20 @@
+name: motoko-send-http-get
+on:
+  pull_request:
+    paths-ignore:
+      - motoko/send-http-get/**
```

Needs to match the actual path
```suggestion
      - motoko/send_http_get/**
```